### DISCUSSION TITLE
Sop falls into howto ?

### DISCUSSION THREAD

**Discussion by rrg92** [evildmp/diataxis-documentation-framework#63] [Q&A]
Hi!

A simple question: Standard Operating Procedures (or template checklists for some technical operation) can fall into How-To guides?

**Comment by anarcat** [comment]
yeah, it's where I typically put those myself. if it's really trivial, it might even land in Tutorial. it really depends on your audience: in my case, documentation is aimed at two very different audiences and I typically use the guide/tutorial distinction to split between those two audiences, but that might not be canon here. :)
